endpoints.json

```diff
@@ -62,5 +62,24 @@
         }
       ]
     }
+  },
+
+  "PATCH /api/articles/:article_id": {
+    "description": "update vote count for article specified by id",
+    "queries": [],
+    "exampleResponse": {
+      "article": [
+        {
+         "article_id": 1,
+         "title": "'Living in the shadow of a great man'",
+         "topic": "mitch",
+         "author": "butter_bridge",
+         "body": "I find this existence challenging",
+         "created_at": "2020-07-09T20:11:00.000Z",
+         "votes": 50,
+         "article_img_url": "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
+        }
+      ]
+    }
```

Comment:
This is really good, but dont forget to include an exampleBody as well of what we expect a user to send to us to be able to increment the votes.